trueos-handbook/introducing.rst

```diff
@@ -718,55 +770,104 @@ A virtualized environment allows a user to test drive an operating
-   background script automatically gives the user account the
-   permissions required to run this application. This might break any
-   existing shortcuts to VirtualBox. To fix the shortcut, logout and in
-   again.
-
-If the computer is running another operating system, download the binary
-for the specific operating system from the
-`VirtualBox Downloads page <https://www.virtualbox.org/wiki/Downloads>`_.
-VirtualBox runs on Windows, Linux, Macintosh, and OpenSolaris. It
-supports a large number of virtual machine installable operating
-systems.
-
-This section describes how to prepare VirtualBox for an installation of
-|trueos| using an :file:`.iso` file.
-
-.. index:: create virtual machine for an iso
-.. _Creating a Virtual Machine for an ISO File:
-
-Creating a Virtual Machine for an ISO File
-------------------------------------------
-
-Once the |trueos| ISO is downloaded and VirtualBox installed on the
-current system, create a virtual machine and use the ISO to install
-|trueos| into the virtual machine. The virtual machine must meet several
-minimum requirements and this section will demonstrate how to configure
-these:
-
-* 1024 MB base memory size.
-
-* A virtual disk **at least 20 GB in size** for a server installation or
-  **at least 50 GB in size** for a desktop installation.
+   background script automatically gives the user account that started
+   VirtualBox the permissions required to run the application. This
+   might break existing shortcuts to VirtualBox. To fix the shortcut,
+   logout and in again.
+
+.. TODO: This section about VirtualBox on other OS' should be removed.
+   Information about other host systems is not necessary or even needed
+   here.
+   Original text;
+   If the computer is running another operating system, download the binary
+   for the specific operating system from the
+   `VirtualBox Downloads page <https://www.virtualbox.org/wiki/Downloads>`_.
+   VirtualBox runs on Windows, Linux, Macintosh, and OpenSolaris. It
+   supports a large number of virtual machine installable operating
+   systems.
+
+.. index:: creating a virtual machine for a |trueos| install
+.. _Creating a Virtual Machine for a |trueos| install:
+
+Creating a Virtual Machine for a |trueos| install
+-------------------------------------------------
+
+How to prepare VirtualBox for an installation of |trueos| using an
+:file:`.iso` file.
+
+Once a |trueos| ISO is <downloaded `https://www.trueos.org/downloads/downloads>`_
+and VirtualBox installed on the host system, create a new virtual
+machine to install |trueos| as a guest OS. The virtual machine must meet
+several minimum requirements in order to be useable. This section will
+demonstrate how to configure the virtual machine for a |trueos| guest.
+
+* A minimum of 1024MB of memory.
+
+* A virtual disk of **at least 20 GB in size** for a server installation
+  or **at least 50 GB in size** for a desktop installation.
```

Comment:
Recommended minimum sizes are quite a bit lower now.
10-15 GB for server and 20-30 GB for desktop (minimums depending on how much the user wants to install new stuff into the VM).

Reply:
updated the sizes and reworked the text a bit.